Documentation/How to Prepare a Publish Profile.md

```diff
@@ -0,0 +1,64 @@
+# How to Prepare a Publish Profile
+
+In order to deploy applications to a development board (e.g. Raspberry Pi, HummingBoard, etc.), you need to setup a Publish Profile.  Below provides a few steps to quickly get started wihin your projects.
+
+1. Once you have an application setup in Visual Studio, right-click the project in the Solution Explorer and select Publish...
+
+2. In the Pick a publish target dialog, select Folder, choose a folder to publish your application files and click Create Profile.  The example below shows a path of C:\PublishedApps.
+
+3. A default profile, FolderProfile.pubxml, will now be created and added to your project.  You can view it in the Solution Explorer under your project > Properties > PublishProfiles folder.
+
+4. It is a good practice to rename your profile to something you can relate with your project.  In the Publish Window, click the Actions dropdown and select Rename Profile.  A Rename Profile dialog prompts you to rename your profile.  Click Save after renaming.
+
+5. You can configure the profile's settings by clicking Configure... in the Publish Window.  A Profile Settings dialog prompt includes a few options.  
+  
+    **Notes**  
+    * Currently, the Target Runtime doesn't offer a selection for Linux ARM.  You will need to manually open the profile's XML and change the RuntimeIdentifier element to **linux-arm** shown below:
```

Comment:
Does `win-arm` work? I thought you would have to manually specify that one as well.

Reply:
To be honest, I have not tried yet.  I just received my other board and have not setup for Win IoT yet.  I will try to do this over the weekend.

Reply:
I have tried it with windows in the past, and what you need is to have:
```xml
<PropertyGroup>
  <RuntimeIdentifiers>linux-arm;win-arm</RuntimeIdentifiers>
</PropertyGroup>
```

Reply:
Thx.  Updated doc.